.gitlab/source_test/linux.yml

```
@@ -2,8 +2,8 @@
 .rtloader_tests:
   stage: source_test
   before_script:
-    - !reference [.retrieve_linux_go_deps]
     - source /root/.bashrc && conda activate $CONDA_ENV
```

```suggestion
    - source /root/.bashrc
```
as conda is launched in the .bashrc, correct?

---

It's true that we use Python 3 everywhere, which is why `source /root/.bashrc`, which activates the `ddpy3` environment, is usually enough.

This is the only case in the pipeline where `conda activate $CONDA_ENV` does need to be specified, because here `$CONDA_ENV` can be either `ddpy2` (if we want to run tests with Python 2, see the `tests_deb-x64-py2` job definition) or `ddpy3` (if we want to run tests with Python 3, see the `tests_deb-x64-py3` definition). The reason for that is that we want to be able to build `rtloader`, the bridge between our go code and the Python interpreter, against both Python 2 and Python 3, and run its unit tests against both.